Husky.java

```diff
@@ -0,0 +1,32 @@
+import java.awt.*;
+
+public class Husky extends Critter {
+    /* @fn getMove
+     * Infects if non-Husky in front of it
+     * Turns right if there's a wall in front or a neighbor
+     * Else hops
+     */
+    public Action getMove(CritterInfo info) {
+        if (info.getFront() == Neighbor.OTHER) {
+            return Action.INFECT;
+        } else if (info.getFront() == Neighbor.WALL || info.getFront() == Neighbor.SAME) {
+            return Action.RIGHT;
+        } else {
+            return Action.HOP;
+        }
+    }
+
+    /* @fn getColor
```

Comment:
this whole comment is pretty redundant.

also I never do @fn <func name> ever, cause that's also redundant.